### PR TITLE
Use non-jwt token

### PR DESCRIPTION
**Pull request by sadlil**

### CODE REVIEW

File: pkg/server.go [hunk 1]
```diff
@@ -505,6 +480,13 @@ func (server *LoginServer) issueIDToken(sub string, w http.ResponseWriter, req *
 	state.Decode(encodedState)
 
 	if state.RedirectTo != "" {
+		// Store in DB
+		token, err := server.store(getNamespace(state.RedirectTo), user)
```
Comment (sadlil) [inline pkg/server.go]: Getting namespace name from `RedirectTo`. 
Is this the right? @TamalSaha 
this url must needs to add at `--allowed-redirect-urls` and in `google console`.

File: pkg/server.go [hunk 2]
```diff
@@ -528,77 +510,84 @@ func (server *LoginServer) issueIDToken(sub string, w http.ResponseWriter, req *
 
 func (server *LoginServer) Run() {
 	// setupSignalHandlers()
+	storage.Init()
```
Comment (sadlil) [inline pkg/server.go]: This inits `dealer` storage. And stores the data in delear user database. this is ok?

File: pkg/server.go [hunk 3]
```diff
@@ -528,77 +510,84 @@ func (server *LoginServer) issueIDToken(sub string, w http.ResponseWriter, req *
...
-	//user.LoginCount = 1
-	//user.LastLogin = time.Now()
-	//_, err = ctx.Store.Engine.Insert(user)
-	//if err != nil {
-	//	return err
-	//}
-	return nil
-}
+func (server *LoginServer) store(ns string, user *storage.User) (string, error) {
+	store, err := storage.NewByName(ns)
```
Comment (sadlil) [inline pkg/server.go]: NewByName(ns) -> ns extracted from redirctURl.

File: pkg/server.go [hunk 4]
```diff
@@ -528,77 +510,84 @@ func (server *LoginServer) issueIDToken(sub string, w http.ResponseWriter, req *
...
+		log.V(10).Infoln("Sending OAuth Token", token)
+		return token, nil
+	}
+	log.V(3).Infoln("User not created create user with", *user)
+	bt, err := contexts.NewBrainTree(store)
+	if err != nil {
+		return "", err
+	}
+	firstName, lastName := parseRealName(user.RealName)
+	customer, err := bt.Customer().Create(&braintree.Customer{
```
Comment (sadlil) [inline pkg/server.go]:
```
Tamal Saha·6:01 PM
gatepass should not call braintree directly
it should call something in dealer that deals with payment Provider
```
Call something in dealer means api calls via grpc?

File: pkg/server.go (hunk 2 above)
Comment (TamalSaha) [inline pkg/server.go]: sgtm.

File: pkg/server.go (hunk 4 above)
Comment (TamalSaha) [inline pkg/server.go]: This should be at least some method call or calling the grpc version of create user.

File: pkg/server.go (hunk 1 above)
Comment (TamalSaha) [inline pkg/server.go]: I need to think more.

**Pull request merged**